.github/settings.yml

```diff
@@ -0,0 +1,57 @@
+repository:
+  # See https://github.com/apps/settings for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: project-template
+
+  # A short description of the repository that will show up on GitHub
+  description: MDN Web Docs Project Template
+
+  # A URL with more information about the repository
+  homepage: https://github.com/mdn/project-template
+
+  # The branch used by default for pull requests and when the repository is cloned/viewed.
+  default_branch: main
+
+  # This repository is a template that others can use to start a new repository.
+  is_template: true
+
+branches:
+  - name: main
+    protection:
+      # Required. Require at least one approving review on a pull request, before merging. Set to null to disable.
+      required_pull_request_reviews:
+        # The number of approvals required. (1-6)
+        required_approving_review_count: 1
+        # Dismiss approved reviews automatically when a new commit is pushed.
+        dismiss_stale_reviews: true
+        # Blocks merge until code owners have reviewed.
+        require_code_owner_reviews: true
+
+collaborators:
+  - username: schalkneethling
+    permission: admin
+
+  - username: Rumyra
+    permission: admin
+
+  - username: fiji-flo
+    permission: admin
```

Comment:
Was not sure who to add here so, this is temporary and open to review.

Reply:
Let's leave it like this for now